infra/config-store/src/main/java/com/simprints/infra/config/store/local/models/ProjectConfiguration.kt

```diff
@@ -26,4 +30,5 @@ internal fun ProtoProjectConfiguration.toDomain(): ProjectConfiguration =
         consent.toDomain(),
         identification.toDomain(),
         synchronization.toDomain(),
+        customJson?.takeIf { it.isNotBlank() }?.let { JsonHelper.fromJson(it) },
```

Comment:
Maybe it's worth adding a catch to protect against misconfigured JSON coming from Vulcan.

Reply:
It would fail on the network communication step, so the chance is slim. But I will put it just in case of migration issues 👍🏻